README.md

```diff
@@ -374,6 +374,7 @@ Services to securely store your Docker images.
 * [Docket](https://github.com/netvarun/docket) - Custom docker registry that allows for lightning fast deploys through bittorrent by [@netvarun](https://github.com/netvarun/)
 * [Docker Hub](https://hub.docker.com/) provided by Docker Inc.
 * [Docker Registry v2][distribution] - The Docker toolset to pack, ship, store, and deliver content
+* [Europa :heavy_dollar_sign:](https://github.com/Distelli/europa) - Private docker registry with support for image pipelines and  webhooks. By [Distelli](https://github.com/Distelli)
```

Comment:
Thank You @Moshe-Immerman - if you replace Distelli by  `@`Distelli that would be awesome (and squash or amend commit) :smiley: